src/Feature.php

```diff
@@ -21,6 +21,24 @@ public function __construct()
         $this->instance = Cache::get(Feature::FEATURE_FLAG_CACHE_KEY, []);
     }
 
+    /**
+    * Check if a feature flag exists.
+    *
+    * @param string $featureKey
+    * @param mixed $variant (optional)
+    * @return bool
+    */
+    public function exists($featureKey)
+    {
+        $featureFlag = FeatureFlag::where('key', $featureKey)->first();
+
+        if (isset($featureFlag)) {
+            return true;
+        }
+
+        return false;
```

Comment:
Let's use the `$this->instance` instead which is all the cached keys, so we avoid making extra queries.


```suggestion
        return isset($this->instance[$featureKey]);
```